workflows/diagnostics/fv3net/diagnostics/offline/compute.py

```diff
@@ -87,14 +87,16 @@ def _get_parser() -> argparse.ArgumentParser:
     parser.add_argument(
         "--evaluation-grid",
         type=str,
-        default=None,
+        default=EVALUATION_RESOLUTION,
         help=(
-            "Optional path to a grid data netcdf that sets the grid resolution at "
-            "which the diagnostics will be computed. If not provided, evaluation grid "
-            "resolution will be C48. If validation data resolution is higher than "
-            "the evaluation grid resolution by an integer multiple, the validation "
-            "data will be coarsened to the evaluation grid resolution. Otherwise the "
-            "evaluation grid resolution must match the validation data resolution."
+            f"Optional arguent that sets the grid resolution at which the diagnostics"
```

Comment:
Specify the required string format in this docstring, eg "c12"?

Reply:
Thanks -- added.